contents/blog/the-posthog-array-1-43-0.md

```diff
@@ -0,0 +1,123 @@
+
+### New: Feature flag variant overrides
+![flag overrides](../images/blog/array/1-43-0-feature-override.gif)
+
+Ever created a multivariate feature flag, and wanted to show the control variant to specific users? Or, maybe you found a new cohort you'd like to add to the test variant? Now you can!
+
+1.43.0 adds the ability to manually override variants via the flag edit screen. It's great for ensuring users get the intended experience, but is also useful for testing as it enables you to deterministically choose a variant, and test across client-side and server-side feature flags.
+
+### Improved: More experiment variants and improved flow
+A regular complaint used to be that, if you made a mistake while making an experiment, you couldn't go back to fix it. Well, we fixed that so you can now:
+
+1. Restart a running experiment while keeping everything the same and using the same feature flag key.
+2. Change the main metric on a running experiment.
+3. Change the secondary metrics on a running experiment.
+
+Finally, as a bonus, you can now have up to nine variants in an experiment, instead of the usual three!
+
+### New: Export recordings to file
+![export recordings](../images/blog/array/1-43-0-export.gif)
+
+Sometimes a recording can be so insightful, so important, that you think - "I want to keep a record of this forever." Well, now you can export any recording to a file and load it back into PostHog for playback in the future. Whether it is to commemorate your first sale or for compliance reasons, the tools are now in your hands.
+
+### New: Role-based access for feature flags
+You can now create roles and group team members together, along with customizing feature flags access for team members! Having access control helps reduce accidental changes and ensures confidence when shipping a new feature.
+
+### Improved: Recording playback controls 
+![recording controls](../images/blog/array/1-43-0-seekbar.gif)
+
+We've revamped the recording seekbar to make it easier to find user activity at a glance, to show where tracked events occur and display what time you're scrubbing to.
+
+### Other improvements & fixes
+You think that's it? Not by a long shot! Version 1.43 also adds hundreds of other improvements and fixes, including...
+
+- **Improvement:** [We've consolidated container build workflows and made changes to build and push multi-arch images](https://github.com/PostHog/posthog/pull/13543).
+- **Improvement:** We made [a simpler, faster process for querying the recently viewed insights on your home page](https://github.com/PostHog/posthog/pull/13529)
+- **Improvement:** We've switched to use `pnpm` instead of `yarn` to [manage dependencies](https://github.com/PostHog/posthog/pull/13190).
```

Comment:
I would put No. 1 and No. 3 at the bottom of the list, because they're honestly completely irrelevant to any PostHog user. It's just back office enhancements for PostHog engineering.